client/src/views/About.vue

```diff
@@ -56,6 +56,18 @@
           <v-card v-else-if="!!game_over">{{game_over}} has won the game!</v-card>
 
           <v-card v-else :class="'ma-3 pa-6'" outlined tile>Waiting for {{ current_player }}</v-card>
+
+          <!-- Organize Cards -->
+          <v-card v-if="username == current_player" :class="'ma-3 pl-6 pa-4'" outlined tile>
```

Comment:
Can you include a screenshot showing these changes?

Reply:
![image](https://user-images.githubusercontent.com/48926044/89112877-e7248f00-d426-11ea-9dea-50b8111f81f2.png)